api/swagger.yaml

```diff
@@ -91,18 +99,43 @@ tags:
   - name: Event
     description: |
       An event is something occurring at a specific time (whether in the past, present, or past alike) and (optionally) at a specific location. Events
-      are the core of this API. Events link all static objects together. The sections below document the available event types. Events are devided
-      into categories:
-      - Update events
-      - Warning events
-      - Action events
-      - Time window events
-      - Associative events
-      - Meta events
-
-      See the respective sections below.
-
-      ## Update events
+      are the core of this API. Events link all static objects together. There are several different event types defined in OpenTripModel. The
+      sections below document the available event types. Events are devided into categories:
+      - [Update events](#section/update-events)
+      - [Warning events](#section/warning-events)
+      - [Action events](#section/action-events)
+      - [Time window events](#section/time-window-events)
+      - [Associative events](#section/associative-events)
+      - [Meta events](#section/meta-events)
+
+      ## <a section="section/event-endpoints"></a>Event endpoints
+      Since events are related to at least one other entity, events can be accessed both via the related entities as well as via a dedicated `events`
+      endpoint. The following table summarizes all available ways to access events via OpenTripModel endpoints:
+
+      | #&#160;events | access          | read/write |                                                                         |
+      | ------------- | --------------- | ---------- | ----------------------------------------------------------------------- |
+      | list          | directly        | read       | [**GET** `/events`](#tag/Event%2Fpaths%2F~1events%2Fget)                |
+      | list          | directly        | write      | [use a Bundle](#tag/Bundle)                                             |
+      | list          | via&#160;entity | read       | **GET** `/{entity}/{uuid}/{lifecycle}/events/`                          |
+      | list          | via&#160;entity | write      | [use a Bundle](#tag/Bundle)                                             |
+      | single        | directly        | read       | [**GET** `/events/{uuid}`](#tag/Event%2Fpaths%2F~1events~1{uuid}%2Fget) |
+      | single        | directly        | write      | [**PUT** `/events`](#tag/Event%2Fpaths%2F~1events%2Fput)                |
+      | single        | via&#160;entity | read       | N/A                                                                     |
+      | single        | via&#160;entity | write      | **PUT** `/{entity}/{uuid}/{lifecycle}/events/`                          |
+
+      ## <a section="section/involved-objects"></a>Involved objects
+      Each event has a list of `involvedObjects`. Each event should have at least one other entity in the list of `involvedObjects`, since entities
```

Comment:
Is this true, are there no events which only belong to other events?

Reply:
There are events that only have another event as `involvedObject`. However, since `Events` are entities as well, the statement still holds true. I'll add a clarification anyway.